auth/server/src/main/java/com/seagates3/service/RequestorService.java

```diff
@@ -63,6 +65,23 @@ static Requestor getRequestor(ClientRequestToken clientRequestToken)
         try {
             perf.startClock();
 
+            GlobalData dataObj =
```

Comment:
skip this block if cachetimout value is zero

Reply:
Fixed